prowler/providers/aws/services/sagemaker/sagemaker_service.py

```diff
@@ -16,12 +16,15 @@ def __init__(self, provider):
         self.sagemaker_notebook_instances = []
         self.sagemaker_models = []
         self.sagemaker_training_jobs = []
+        self.endpoint_configs = {}
         self.__threading_call__(self._list_notebook_instances)
         self.__threading_call__(self._list_models)
         self.__threading_call__(self._list_training_jobs)
-        self._describe_model(self.regional_clients)
-        self._describe_notebook_instance(self.regional_clients)
-        self._describe_training_job(self.regional_clients)
+        self.__threading_call__(self._list_endpoint_configs)
+        self.__threading_call__(self._describe_model)
+        self.__threading_call__(self._describe_notebook_instance)
+        self.__threading_call__(self._describe_training_job)
+        self.__threading_call__(self._describe_endpoint_config)
```

Comment:
For describe functions, we have to pass the object that we are going to iterate since it is not going to be the regional_clients attribute.